infra/build/developer-tools/build/scripts/test_lint.sh

```diff
@@ -35,15 +50,55 @@ tests=(
   check_terraform
 )
 
+# parse args
+for arg in "$@"
+do
+  case $arg in
+    -m|--markdown)
+      MARKDOWN=1
+      shift
+      ;;
+    -c=*|--contrib-guide=*)
+      CONTRIBUTING_GUIDE="${arg#*=}"
+      shift
+      ;;
+      *) # end argument parsing
+      shift
+      ;;
+  esac
+done
+
 for test in "${tests[@]}"; do
-  if ! "${test}"; then
+  # if not in markdown mode, pipe test output to stdout tty
+  # nested if condition is a workaround for test[[]] not echoing some outputs from check_* tests even with subshell
+  if [[ $MARKDOWN -eq 0 ]]; then
+    if ! "${test}"; then
```

Comment:
I couldn't find a better way to do this, when I tried using `[[  $MARKDOWN -eq 0 & $(${test}) ]] `and various other combinations, it does not print all the output of `$(${test})`